src/pages/shared.js

```diff
@@ -0,0 +1,16 @@
+import "../style/index.css";
+import Footer from "../component/footer";
+import Header from "../component/header";
+import Folder from "../main/folder";
+
+function Shared() {
+  return (
+    <div>
+      <Header />
+      <Folder />
+      <Footer />
+    </div>
```

Comment:
```suggestion
    <>
      <Header />
      <Folder />
      <Footer />
    </>
```
Shared 컴포넌트를 보고 바로 구조가 그려지는 이 구조는 굉장히 좋네요 ㅎㅎ
단순히 컴포넌트를 묶는 것으로 사용한다면 div 보단 fragment 태그를 사용하는 것이 좋을 것 같네요 ㅎㅎ